README.md

```diff
@@ -23,17 +23,34 @@ In short: JSON documents with named merkle-links that can be traversed.
 
 ## Repositories
 
-IPLD is a multifaceted, distributed effort. Here are our current repositories:
+IPLD is a multifaceted, distributed effort.
 
+### Specifications
+
+- [specs](https://github.com/ipld/specs) - The specifications for IPLD
 - [cid](https://github.com/ipld/cid) - Self-describing content-addressed identifiers for distributed systems
+
+### Implementations
+
+- [**JavaScript:** js-ipld-dag-cbor](https://github.com/ipld/js-ipld-dag-cbor) - JavaScript implementation of the IPLD spec.
```

Comment:
@RichardLitt You are missing that `IPLD` is not a spec for a single implementation, but rather a:
- data format description that can be followed by multiple implementations (dag-cbor, dag-pb, etc)
- and interface definition (interface-ipld-format)
- a tool to resolve paths  (ipld-resolver)